### ISSUE TITLE
Add generation time per function call

### ISSUE THREAD

**Issue by erikdemarco** [johnbillion/query-monitor#617]
Scalability is hurt by slow page generation.

It would be great if the query monitor plugin could report slow function as well. 

Something like this:
https://wordpress.org/plugins/debug-functions-time/

Yes there is already plugin for that, but this plugin seems not seriously built. A bit buggy.

**Comment by johnbillion** [comment]
Yes I'm planning on implementing this in a similar manner to how [the wp profile command for WP-CLI](https://github.com/wp-cli/profile-command) does it. No ETA for now though.
